packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -418,4 +418,9 @@ class PluginInterfaceResolution {
       'dartClass': plugin.pluginDartClassPlatforms[platform] ?? '',
     };
   }
+
+  @override
+  String toString() {
+    return '<PluginInterfaceResolution ${plugin.name} for $platform>';
```

Comment:
I added this to help with debugging, and it seemed harmless to keep it for future debugging.